documentation/latency-debugging-guide.md

```diff
@@ -0,0 +1,257 @@
+# Spanner JDBC Driver - Latency Debugging Guide
+
+The Spanner JDBC driver supports OpenTelemetry tracing. Tracing can be used to investigate slow
```

Comment:
We also have metrics which is helpful in debugging latency issues. Should that info also be added here https://cloud.google.com/spanner/docs/latency-points

Reply:
Added a link to the page. The intention is to also add this guide to the documentation (probably to the same submenu)